openquake/hazardlib/gsim/parker_2020.py

```diff
@@ -399,6 +400,11 @@ def __init__(self, region=None, saturation_region=None, basin=None,
         else:
             self.saturation_region = saturation_region
         self.basin = basin
+        self.usgs_basin_scaling = usgs_basin_scaling
+        if (self.usgs_basin_scaling and 'z2pt5' not in
+            self.REQUIRES_SITES_PARAMETERS):
+            raise ValueError('User must specify a GSIM class for this GMPE '
```

Comment:
Maybe the error message came from the ModifiableGMPE? It does not make sense here.

Reply:
The user might specify a  to use the US basin scaling, but specify a class of this GMM which does not consider basin effects (especially for this gsim where there is `ParkerEtAl2020SInterface` and `ParkerEtAl2020SInterB`, and so I think a warning might be a good idea to highlight this?